securitas/security/ipa.py

```diff
@@ -11,7 +11,10 @@ def parse_group_management_error(data):
 
     TODO: send this upstream.
     """
-    failed = data['failed']
+    try:
+        failed = data['failed']
+    except KeyError:
+        return
```

Comment:
This new branch will probably need a unit test, sorry... ;-)

Reply:
added a test for this one.